hmri_create_MTProt.m

```diff
@@ -140,7 +140,11 @@
 % dimension defined below (dm) is used across the whole script. It must not
 % been redefined.
 V_pdw = spm_vol(mpm_params.input(PDwidx).fnam);
-dm = V_pdw(1).dim;
+
+%set a reference space to be used for different contrasts
+%we set it to PDw
+V_ref = V_pdw(1);
+dm = V_ref.dim;
```

Comment:
Thanks for implementing this change. However I think it would also be a good idea to move the comment from lines 139--141 to here, merging your comment with the old one.